### DISCUSSION TITLE
Use the gp star wand board

### DISCUSSION THREAD

**Discussion by seattlebuster** [gpstar81/GPStar-proton-pack#147] [Q&A] [answered]
Is it theoretically possible to use the gp star wand board and still wire it up to communicate with a mega in the main pack?   I purchased everything to build the hard wire kit back early this year. My first pack is done, works great. All hard wired with the nano and mega.  

I don't really want to do the wand again with the nano. That really sucked and was super tight on space.   Can I just supply the tx, rx , power and ground from the pack to the new board design and have it work? Wiring in the wav board also. 

I don't mind the mega in the pack since there is space.

Just curious since the new board design is a massive space saver.

**Comment by gpstar81** [comment, marked as answer]
> Is it theoretically possible to use the gp star wand board and still wire it up to communicate with a mega in the main pack?   I purchased everything to build the hard wire kit back early this year. My first pack is done, works great. All hard wired with the nano and mega.  
> 
> 
> 
> I don't really want to do the wand again with the nano. That really sucked and was super tight on space.   Can I just supply the tx, rx , power and ground from the pack to the new board design and have it work? Wiring in the wav board also. 
> 
> 
> 
> I don't mind the mega in the pack since there is space.
> 
> 
> 
> Just curious since the new board design is a massive space saver.

Yes you can intermix them.
